kogito-integration-tests/kogito-quarkus-integration-test/javax.inject.Named

```diff
@@ -0,0 +1,2 @@
+io.quarkus.it.kogito.jbpm.OrderItemsProcess
+io.quarkus.it.kogito.jbpm.OrdersProcess
```

Comment:
This shouldn't be in.

It looks like an issue we had in the past with sisu-inject being a dependency and going berserk. Can you check?

See https://github.com/quarkusio/quarkus/pull/2889 and https://github.com/quarkusio/quarkus/pull/2986

Reply:
@gsmet right, I'll give a look